### PR TITLE
issue/1 Added jsx support

### LINKED ISSUE #1: Will not work with jsx

### PR DESCRIPTION
**Pull request by oliverfoster**
fixes #1 

### Fixed
* Remove reliance on handlebars render

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ The attributes listed below are used in *course.json* to configure **Adapt Graph
 >**\_onPauseRewind** (Boolean): Rewind when the pause button is clicked. Defaults to `false`.
 
 ----------------------------
-**Version number:**  0.0.17<br />
-**Framework versions:**  >=5.8<br />
+**Version number:**  0.1.0<br />
```
Comment (swashbuck) [inline README.md]: @oliverfoster I believe we've been removing the version number from the README.md docs. That way, we don't have to keep it updated when we add the automated release stuff.

**Comment by swashbuck** [comment]
> _fileExtension (String): The image file extension of the json file. Acceptable values are mp4, tiff, jpe. Defaults to mp4.

I think this could use some clarification. It took me awhile to figure out how to configure it.

1. "The image file extension of the json file." Could be reworded to something like "The video file extension to use. You can use files of this type in place of images in JSON files."
2. "Acceptable values are mp4, tiff, jpe" If the purpose is to replace _image_ elements with _videos_, why would we allow TIFF and JPG? I would think maybe `mp4` and `webm` would be allowed. Side note: shouldn't it be 'jp**g**' not 'jpe'?
3. In __adapt-graphicVideo.js_, `_fileExtension` defaults to 'svgz'. However, in VideoView.js, it defaults to 'mp4'. Are svgz files allowed? And should these defaults be the same?

Then, maybe _example.json_ includes a JSON example from a core component like Graphic?

```
    // components.json - Example using mp4 videos in a Graphic component
    {
        "_id": "c-05",
        "_parentId": "b-05",
        "_type": "component",
        "_component": "graphic",
        "_classes": "",
        "_layout": "left",
        "title": "Graphic component",
        "displayTitle": "",
        "body": "",
        "instruction": "",
        "_graphic": {
            "large": "course/en/images/graphic.mp4",
            "small": "course/en/images/graphic.mp4"
        }
    }
```
These documentation changes can also apply to [adapt-graphicLottie](https://github.com/cgkineo/adapt-graphicLottie).

File: README.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ The attributes listed below are used in *course.json* to configure **Adapt Graph
...
+**Framework versions:**  >=5.14<br />
```
Comment (oliverfoster) [inline README.md]:
```suggestion
```

**Comment by oliverfoster** [comment]
> 1. "The image file extension of the json file." Could be reworded to something like "The video file extension to use. You can use files of this type in place of images in JSON files."

Done

> 2. "Acceptable values are mp4, tiff, jpe" If the purpose is to replace _image_ elements with _videos_, why would we allow TIFF and JPG? I would think maybe `mp4` and `webm` would be allowed. Side note: shouldn't it be 'jp**g**' not 'jpe'?

Just mp4 is fine.

> 3. In __adapt-graphicVideo.js_, `_fileExtension` defaults to 'svgz'. However, in VideoView.js, it defaults to 'mp4'. Are svgz files allowed? And should these defaults be the same?

Both plugins need file extensions that are different from one another and which appear in the correct view of the AAT asset picker and which are the correct mime type to get through apache and php, and which are not standard image file extensions. This plugin replaces standard `<img` tags with a `<video` tag where the specified file extension is detected, using `jpg` would mean that the plugin would try to video play all normal jpgs.

**Comment by swashbuck** [comment]
> I think this could use some clarification. It took me awhile to figure out how to configure it.

@oliverfoster Let me know if you would like me to create a documentation improvement ticket.

**Pull request merged**